### DISCUSSION TITLE
Running test suite with phpunit

### DISCUSSION THREAD

**Discussion by mega** [php/frankenphp#891] [Q&A]
Can we run phpunit tests on a frankenphp container with php-cli option? 

I've tried and it returned this message:

```
/srv/app # frankenphp php-cli bin/phpunit tests/
You need to set up the project dependencies using Composer:

    composer install

You can learn all about Composer on https://getcomposer.org/.
```

I'm using the `dunglas/frankenphp:latest-php8.2-alpine` image but it seems it's not up-to-date with the latest release 1.2.1
FrankenPHP container version
```
/srv/app # frankenphp --version
FrankenPHP v1.1.5 PHP 8.2.19 Caddy v2.7.6 h1:w0NymbG2m9PcvKWsrXO6EEkY9Ru4FJK8uQbYcev1p3A=
```

**Comment by mega** [comment]
So, I rebuilt the vendor directory (with a clean composer install) and was able to run `bin/phpunit` but now I get a new error:

```
/srv/app # frankenphp php-cli bin/phpunit tests/
PHPUnit 9.6.19 by Sebastian Bergmann and contributors.

Unknown option "--ansi"
```

Any ideas?

**Reply by AlliBalliBaba** [reply]
Does running tests with the regular php cli work? Aka:
`bin/phpunit tests/` or just `vendor/bin/phpunit`.
Updating Phpunit might also solve the issue, newest version is 11.

**Reply by mega** [reply]
I'm using this phpunit version so that I can check the symfony deprecation warnings. Not sure they already bumped it for v11.
As far as I understand `--ansi` is an option for composer, so I'm not sure what this means.
Nevertheless, the latest frankenphp image I'm using is not up-to-date. Maybe that's related?

Thank you for your reply

**Reply by AlliBalliBaba** [reply]
Can you provide the content of the `bin/phpunit` file? When running the regular phpunit file  from the vendor folder (`vendor/bin/phpunit`) it works for me at least with `frankenphp php-cli`. Updating frankenphp to 1.2.1 might also help, though the error looks like it's coming from some script that adds `--ansi` to the end of the phpunit command.
